src/components/app/ProfileViewer.js

```diff
@@ -1,18 +1,20 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+/* this source code form is subject to the terms of the mozilla public
```

Comment:
lol, I think some some lowercasing went wild in vim.

Reply:
oops